README.rst

```diff
@@ -14,6 +14,63 @@ Contributing
 Please read `CONTRIBUTING.md <https://github.com/aws/sagemaker-mxnet-training-toolkit/blob/master/CONTRIBUTING.md>`__
 for details on our code of conduct, and the process for submitting pull requests to us.
 
+Testing
+-------
+
+Set up a virtual environment for testing.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+One of the multiple ways to setup a virtual environment
+
+::
+
+   # use a package virtualenv
+   # create a virtualenv
+   virtualenv -p python3 <name of env>
+   # activate the virtualenv
+   source <name of env>/bin/activate
+
+Install requirements
+~~~~~~~~~~~~~~~~~~~~
+
+::
+
+   pip install --upgrade .[test]
+
+
+Test locally
+~~~~~~~~~~~~
+
+To run specific test
+
+::
+
+   tox -- -k test/unit/test_training.py::test_train_for_distributed_scheduler
+
+To run an entire file
+
+::
+
+   tox -- test/unit/test_training.py
+
+To run all tests within a folder [e.g. integration]
+
+::
+
+   tox -- tests/integ
```

Comment:
integ tests cannot be run without building images

Reply:
Adding -B would trigger image build and should be enough for local mode tests, but sagemaker tests an AWS account needs to be specified and correct ECR repo need to be specified.

Reply:
Updated it. Take a look @nadiaya @ChuyangDeng